src/components/functional_module/include/functional_module/timer/timer_director_impl.h

```diff
@@ -113,6 +113,7 @@ void TimerDirector::UnregisterTimer(const ModuleTimer<T>& timer) {
     return;
   }
   it->second->stop();
```

Comment:
It is called in destructor:
Thread::~Thread() {
  finalized_ = true;
  stopped_ = true;
  join();
  pthread_join(handle_, NULL);
}